README.md

```diff
@@ -229,41 +120,18 @@ Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-
-## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
 > Give credit to everyone who inspired your codebase.
 
-I would like to thank...
+I would like to thank Microverse Inc.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FAQ (optional) -->
-
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
```

Comment:
It's great that you used the Microverse template 👍🏽 but one thing to consider is having the right `MIT.md` link in your `README.md` file 📄 , your current link 🔗 is broken, you can consider one of the following options:
- [ ] Create a `LICENSE` file and add the content of your `MIT.md` inside of it, or just change your `MIT.md` file name to become `LICENSE`
- [ ] Edit your `README` to reference `MIT.md` instead of `LICENSE`

I hope this helps, thank you 😉

Reply:
Hi @didierganthier, thank you for making time to review my PR. The required changes you mentioned has been made, kindle re-review my PR for me. Thank you.

Reply:
Hey @Cosmosha, you have to request another review on the dashboard and make sure you also address the changes I mentioned in the summary 🙂 the PR you submit must includes changes in all the files related to the project, but this one doesn't. For example, I couldn't review your `index.html` and `style.css` file because they aren't part of this PR, make sure you address that and request a review again on the dashboard, thanks. 😊